core/src/builder/companion/src/expand/mod.rs

```diff
@@ -140,7 +89,10 @@ pub fn expand(def: CompanionDef) -> SynResult<TokenStream> {
 						head: companion.#parachain_names.head(),
 						code: companion.#parachain_names.code(),
 					};
-					companion.#relay_chain_name.onboard_para(para).map_err(|_| ()).map(|_| ())?;
+
+					let collator = companion.#parachain_names.collator();
```

Comment:
I have yet to confirm that this actually works. For now, the project compiles successfully without complaining about missing args to `onboard_para`.